xrtl/port/linux/ui/BUILD

```diff
@@ -44,6 +44,7 @@ cc_test(
         "//xrtl/tools/target_config",
     ],
     size = "small",
+    tags = ["requires_gui"],
```

Comment:
You could enumerate custom tags like this (requires_gui, requires_gpu) somewhere (README or another markdown file?) so there's one place to look when authoring a new test.